README.md

```diff
@@ -1,36 +1,184 @@
 o-brand [![Circle CI](https://circleci.com/gh/Financial-Times/o-brand/tree/master.svg?style=svg)](https://circleci.com/gh/Financial-Times/o-brand/tree/master)
 =================
 
-**⚠️ This is a work in progress ⚠️**
-
-Tools to "brand" Origami components. `o-brand` is intended as an internal tool and is not recommended for use outside of Origami components.
+Tools to "brand" Origami components. Other non-Origami projects should not depend on `o-brand` directly.
 
 - [Terms](#terms)
 - [Usage](#usage)
-	- [Sass](#sass)
 - [Contact](#contact)
 - [Licence](#licence)
 
 ## Terms
 
 ### Brand
 
-A brand is a collection of configuration to tailor a component for specifc usecases. A branded Origami component is able to output the correct visual style and features for a requested brand.
+A brand is a collection of configuration to tailor a component for specific use-cases. A branded Origami component is able to output differing styles for a requested brand. It is also able to include extra styles specific to a brand.
 
 Examples of brands include:
 
 - Master: FT branding for public ft.com sites and affiliates.
-- Internal: A minimal style with subtle branding for internal products, tools, and documentation.
+- Internal: Style suitable for internal products, tools, and documentation.
 - Whitelabel: Base, structural styles only.
 
-### Variants
-
-A branded component may contain a number of variants e.g. a button component may provide a "big" and "small" variant. Variants may modify the appearance or functionality of a component. They may override default brand styles, e.g. change the colour of a button. Variants may also require new markup e.g. to add a breadcrumb to a header component. Variants must be optional and build upon a fully functional branded component.
+### Variant
+
+A branded component may contain one or more variants e.g. a button component may provide a "big" and "small" variant. Variants may change the appearance or functionality of a component. They may override default brand styles, e.g. change the colour of a button. Variants may also require new markup e.g. to add a breadcrumb to a header component. Variants must be optional and build upon a fully functional branded component.
```

Comment:
I think we need to define variant before we use it in a sentence. Perhaps restructuring the paragraph will help?

>Variants are things which can alter the appearance and/or functionality of a component. E.G. A branded component may contain multiple variants which change the appearance of the component, such as a "big" variant and a "small" variant. Etc.